packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/sign-the-csr/index.md

```diff
@@ -2,7 +2,22 @@
 title: Sign the CSR
 ---
 
-Follow the third-party process that your company uses to sign the CSR. You can't move to step four until you complete the process because the signed CSR is required to continue.
+Follow the third-party Certificate Authority (CA) process that your company uses to sign the CSR. You can't proceed to the next step until your certificate is signed using this process.
+
+The following is a list of popular third-party CA providers that can be used to sign the CSR:
```

Comment:
I don't think we want to list this, do we? I allowed this in the SFDC KB article but I'm not sure if we want to put it on the KB site. I defer to PM though if they want to add this. It will be the first time we put CA vendors on the DOC site.

Reply:
@venkatviswanathan-okta, @chrismorris-okta, should we be listing names of specific CAs in d.o.c. content?

Reply:
I don't think we need to list CAs in our doc. We don't need to be opinionated on what CA a customer should use. As long as you generate the CSR from Okta and get it signed by a CA, we should be good.

Reply:
I have no problem listing this out if it useful. I've consistently heard from the developers I spoke with that they want us to provide more guidance and best practices on what they should do and how. Also, we should always create docs for the lowest common denominator. e.g. A front-end developer may not be familiar with this process at all. Just curious, have we gotten any feedback from support or the feedback form regarding this, or is this something we we are proposing on our own? 

My only concern is keeping this section up to date. I don't think we should embed the instructions in our docs, but just list the top 3-4 CAs and link out to their docs for the steps.

Reply:
`e.g. A front-end developer may not be familiar with this process at all.` - This is a SAML App feature and the target persona is an Enterprise IT Administrator. Companies use a wide variety of certificates Authority.